system/timezone.go

```diff
@@ -0,0 +1,61 @@
+package system
+
+import (
+	"log"
+	"os"
+)
+
+// GetTimezone returns the current system time zone
+func GetTimezone() (string, error) {
+	link, err := os.Readlink("/etc/localtime")
+	if err != nil {
+		log.Println("Error finding time zone, ", err)
+		return "", err
+	}
+
+	return link, nil
+}
+
+// SetTimezone sets the current system time zone
+// Takes *US* time zones
+func SetTimezone(zone string) error {
+	if _, err := os.Lstat("/etc/localtime"); err == nil {
+		err := os.Remove("/etc/localtime")
+		if err != nil {
+			log.Println("Error removing old time zone link, ", err)
+			return err
+		}
+	}
+
+	err := os.Symlink("/usr/share/zoneinfo/US/"+zone, "/etc/localtime")
+	if err != nil {
+		log.Println("Error linking to new time zone, ", err)
+		return err
+	}
+
+	return nil
+}
+
+// ReadTimezones returns a list of possible time zones
+// from the system
+// Returns *US* time zones
+func ReadTimezones() (list []string, err error) {
```

Comment:
what do you think of passing in another argument perhaps named zoneInfoDir. This might be:

* "" (read zones in root dir)
* "US" (read zones in US dir)
* "posix/America" (read zones in ....

Reply:
Just thinking of making this useful outside the US.